### PR TITLE
merge

### PR DESCRIPTION
**Pull request by Bevcuk**

### CODE REVIEW

File: data_converter/settings_local.base.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+# DATABASE_CONFIG = {
+#     'ENGINE': 'django.db.backends.sqlite3',
```
Comment (Anntroy) [inline data_converter/settings_local.base.py:5]: Why sqlite3, and not Psql?

File: location_register/converter/koatuu.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import config
-from data_ocean.services.main import Converter
-from data_ocean.models.ratu_models import Region, District, City, Citydistrict
+from data_ocean.converter import Converter
+from location_register.models import Region, District, City, Citydistrict
+
 
 class KoatuuConverter(Converter):
```
Comment (Anntroy) [inline location_register/converter/koatuu.py:6]: Can you, please, uncomment lines: 95, 101, 105, 106, 110. I sow now that I left them commented by error.

File: data_converter/settings_local.base.py (hunk 1 above)
Comment (Bevcuk) [reply data_converter/settings_local.base.py:5]: it is only example file

File: location_register/converter/koatuu.py (hunk 2 above)
Comment (Bevcuk) [reply location_register/converter/koatuu.py:6]: done

File: data_converter/settings_local.base.py (hunk 1 above)
Comment (Anntroy) [reply data_converter/settings_local.base.py:5]: ok

File: location_register/converter/koatuu.py (hunk 2 above)
Comment (Anntroy) [reply location_register/converter/koatuu.py:6]: Thanks

Comment (Anntroy) [reply location_register/converter/koatuu.py:6]: Thanks

**Pull request merged**